int32 nManualInstruction
---
int32 nRetCode
